Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/config/config.go

```diff
@@ -184,7 +184,7 @@ func (s *podStorage) Merge(source string, change interface{}) error {
 		if len(deletes.Pods) > 0 {
 			s.updates <- *deletes
 		}
-		if len(adds.Pods) > 0 {
+		if len(adds.Pods) > 0 || firstSet {
```

Comment:
Hrm, that cherry-pick went terribly wrong.

On Tue, Oct 20, 2015 at 10:56 AM, David Eads <notifications@github.com>
wrote:

> In Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/config/config.go
> <https://github.com/openshift/origin/pull/5222#discussion_r42506291>:
>
> > @@ -184,7 +184,7 @@ func (s *podStorage) Merge(source string, change interface{}) error {
> >  		if len(deletes.Pods) > 0 {
> >  			s.updates <- *deletes
> >  		}
> > -		if len(adds.Pods) > 0 {
> > +		if len(adds.Pods) > 0 || firstSet {
>
> This seems weird, we're triggering a sync by faking an update ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5222/files#r42506291>.
>